go.mod

```diff
@@ -53,3 +53,5 @@ require (
 
 // Workaround for github.com/containerd/containerd issue #3031
 replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
+
+go 1.11
```

Comment:
What's this line for?

Reply:
Snipped from go help go.mod
```
The expected language version, set by the go directive, determines
which language features are available when compiling the module.
Language features available in that version will be available for use.
Language features removed in earlier versions, or added in later versions,
will not be available. Note that the language version does not affect
build tags, which are determined by the Go release being used.
```

When you build with go1.12 or go 1.13 without a go directive it modifies the go.mod to adds it to be whatever you're using. Since we support/CI/test building back to go1.11 I thought it'd be worth adding it to be more explicit about the minimum version we are supporting.

Reply:
I'm not wholly opposed to it, but I don't know that it's important at this point in our development cycle that we stick to a particular version of Go.  If you wanted to move us fully to 1.13 (and set `go 1.13` here), I think that would be fine as well.

Reply:
Considering Go's [promise of compatibility](https://golang.org/doc/go1compat) I don't think we should force users to use 1.13 unless there are strong reasons for it.

Reply:
I don't see this as making a commitment to a particular version of go other than the commitment we've made in the getting started docs already which say go1.11 or later.

I don't feel super strongly about it either way.